src/client/components/body-portal/body-portal.mock.tsx

```diff
@@ -0,0 +1,16 @@
+import { BodyPortal } from './body-portal';
+import * as sinon from 'sinon';
+
+export class BodyPortalMock {
+  static disableBodyPortal() {
+    beforeEach(() => {
+      sinon.stub(BodyPortal.prototype, 'render', function() { return this.props.children; });
+      sinon.stub(BodyPortal.prototype, 'teleport');
+    });
+
+    afterEach(() => {
+      (BodyPortal.prototype.render as Sinon.SinonStub).restore();
+      (BodyPortal.prototype.teleport as Sinon.SinonStub).restore();
+    });
+  }
```

Comment:
Yeah, although what I had mixed feelings about is that it's not super easy to tell from the outside that the `disabledBodyPortal` actually handles the enabling part as well... So there's room for improvement, I guess.